src/cport/modules/predictprotein.py

```diff
@@ -0,0 +1,184 @@
+import logging
+import time
+import sys
+import os
+import zipfile
+import glob
+import pandas as pd
+import tempfile
+import shutil
+
+from selenium import webdriver
+from selenium.webdriver.common.keys import Keys
+from selenium.webdriver.chrome.options import Options
+from selenium.common.exceptions import NoSuchElementException
+
+from cport.url import PREDICTPROTEIN_URL
+from cport.modules import utils
+
+log = logging.getLogger("cportlog")
+
+# Total wait (seconds) = WAIT_INTERVAL * NUM_RETRIES
+WAIT_INTERVAL = 10  # seconds
+NUM_RETRIES = 12
+
+
+class Predictprotein:
+    def __init__(self, pdb_id, chain_id):
+        self.pdb_id = pdb_id
+        self.chain_id = chain_id
+        self.wait = WAIT_INTERVAL
+        self.tries = NUM_RETRIES
+
+    def submit(self):
+        """Makes a submission to the PredictProtein server"""
+        fasta_code = utils.get_fasta_from_pdbid(self.pdb_id, self.chain_id)
+
+        # headless so that browser windows are not visually opened and closed
+        options = Options()
+        options.add_argument("headless")
+
+        # chrome has the most flexibility and ease-of-use
+        driver = webdriver.Chrome(chrome_options=options)
+        driver.get(PREDICTPROTEIN_URL)
+
+        # identifies the textarea
+        elem = driver.find_element_by_id("sequence")
+        elem.clear()
+        elem.send_keys(fasta_code)
+
+        # selects the specific button for submission
+        driver.find_element_by_css_selector(
+            "a.btn.btn-submit.btn-primary.btn-large"
+        ).click()
+
+        # sleep so that the page is properly loaded before continuing
+        time.sleep(5)
```

Comment:
Could this be affected by network speed? Could we think of another way of checking if the page is loaded? Else might be an idea to have this as a global variable `PAGE_LOAD_WAIT` or something like that - since it appears multiple times in the code